compiler/crates/relay-compiler/tests/relay_compiler_integration/fixtures/preloadable_query_typescript.expected

```diff
@@ -53,19 +53,18 @@ export default node;
 
 //- __generated__/fooQuery.graphql.ts
 /**
- * <auto-generated> SignedSource<<461989c03655ebfecf253b741f65ef91>>
+ * <auto-generated> SignedSource<<09465a6feb3ad10a80c2e0606387a0e3>>
  * @relayHash ae6874c86ce5db2df8d6b253a6a0ec13
  * @lightSyntaxTransform
  * @nogrep
  */
 
 /* tslint:disable */
 /* eslint-disable */
-// @ts-nocheck
 
 // @relayRequestID ae6874c86ce5db2df8d6b253a6a0ec13
 
-import { ConcreteRequest, Query } from 'relay-runtime';
+import { ConcreteRequest } from 'relay-runtime';
```

Comment:
I wonder if these fixture changes are actually out of date and come from https://github.com/facebook/relay/pull/4754?

Reply:
Yep, will remove